Menu.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include "Menu.h"
+
+Menu::Menu(Lista<string> ListaPV,Lista<string> ListaPNV,Lista<string> ListaPR) {
+
+
+    this->ListaPV = ListaPV;
+    this->ListaPNV = ListaPNV;
+    this->ListaPR = ListaPR;
```

Comment:
```suggestion
Menu::Menu(Lista<Pelicula> peliculasVistas, Lista<Pelicula> peliculasNoVistas, Lista<Pelicula> peliculasRecomendadas) {

    this->peliculasVistas = peliculasVistas;
    this->peliculasNoVistas = peliculasNoVistas;
    this->peliculasRecomendadas = peliculasRecomendadas;
    
```
Las listas tienen que ser de películas. El cambio en los nombres de las listas es una recomendación para mayor claridad, pero quizás es medio largo.